addons/product_margin/security/ir.model.access.csv

```diff
@@ -1 +1,2 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_report_product_margin,report.product.margin,model_report_product_margin,,1,0,0,0

```

Comment:
@rim-odoo Here create a new model so give the default access rules otherwise it will give the warning.